frontend/src/app/libs/event-creation/event-creation.component.ts

```diff
@@ -1,56 +1,63 @@
 import { Component, OnInit } from '@angular/core';
 import { BreadcrumbFacadeService } from '../common-components/components/breadcrumb/store/breadcrumb.facade';
 import { FormGroup } from '@angular/forms';
-import { NewEvent } from './store/model/model';
-import { NewEventFacadeService } from './store/newEvent.facade';
+import { NewEventFacadeService } from './store/event.facade';
+import { Tabs } from '../common-components/components/tab-navigation/models/tabs';
+import { ITask } from '../common-components/components/tasks-creator/models/task';
 
 @Component({
 	selector: 'app-event-creation',
 	templateUrl: './event-creation.component.html',
 	styleUrls: ['./event-creation.component.scss'],
 })
 export class EventCreationComponent implements OnInit {
-	public currentTab: string = 'generalTab';
-	public newEvent: NewEvent;
-	public formFromGeneralComponent: FormGroup;
+	public currentTab: Tabs = Tabs.generalTab;
+  public tabsEnum: typeof Tabs = Tabs;
+
+	private formFromGeneralComponent: FormGroup;
+	private task: ITask;
+
 	constructor(public breadcrumbFacadeService: BreadcrumbFacadeService, public newEventsFacadeService: NewEventFacadeService) {
 	}
 
 	public ngOnInit(): void {
 		this.breadcrumbFacadeService.loadBreadcrumb('New Event');
 	}
-	public get generalTab(): boolean {
-		return this.currentTab === 'generalTab';
-	}
 
-	public get tasksTab(): boolean {
-		return this.currentTab === 'tasksTab';
+	public changeTab(tab: Tabs): void {
+		this.currentTab = tab;
 	}
 
-	public get invitationsTab(): boolean {
-		return this.currentTab === 'invitationsTab';
-	}
+  public send(isDraft: boolean): void {
+    switch (this.currentTab) {
+      case Tabs.generalTab:
+        this.formFromGeneralComponent.patchValue({
+          isDraft,
+        });
+        this.newEventsFacadeService.createGeneralEvent(this.formFromGeneralComponent.value);
+        break;
 
-	public changeTab(tab: string): void {
-		this.currentTab = tab;
-	}
-	public saveAsDraft(): void {
-		this.formFromGeneralComponent.patchValue({
-			isDraft: true
-		});
-		this.newEventsFacadeService.createNewEvent(this.formFromGeneralComponent.value);
-		console.log(this.formFromGeneralComponent.value);
-	}
+      case Tabs.tasksTab:
+        this.task.isDraft = isDraft;
+        this.newEventsFacadeService.createTask(this.task);
+        break;
 
-	public send(): void {
-		this.formFromGeneralComponent.patchValue({
-			isDraft: false
-		});
-		this.newEventsFacadeService.createNewEvent(this.formFromGeneralComponent.value);
-		console.log(this.formFromGeneralComponent.value);
-	}
+      case Tabs.invitationsTab:
+        break;
+
+      default:
+    }
+  }
 
 	public getFormFromGeneralComponent(data: FormGroup): void {
 		this.formFromGeneralComponent = data;
 	}
+
+  public getTaskCreationForm(task: ITask): void {
```

Comment:
Function name started with get, then this function return something.
Looks like here need to name set..